Marble Blast Platinum/platinum/client/ui/playMissionGui.gui

```diff
@@ -1221,13 +1216,14 @@ function PlayMissionGui::open(%this, %marbleland) {
 		}
 	}
 
+	if (%this.enabledHolidays[$GlobalHoliday] $= "")
+		$this.enabledHolidays[$GlobalHoliday] = true;
```

Comment:
I don't think this will have any negative effects, but if `$GlobalHoliday` is blank then it will set `%this.enabledHolidays = true;`. There aren't any more checks for `%this.enabledHolidays[$GlobalHoliday]` so should be fine